.github/workflows/auto-version-bump.yml

```
@@ -0,0 +1,73 @@
+          if [[ -n "$OLD_GAME_FILE" && -n "$NEW_GAME_FILE" ]]; then
+            mv "$OLD_GAME_FILE" "$NEW_GAME_FILE"
+            find . -type f -exec sed -i "s/$OLD_GAME_FILE/$NEW_GAME_FILE/g" {} +
+          fi
+
+          if [[ -n "$OLD_STYLE_FILE" && -n "$NEW_STYLE_FILE" ]]; then
+            mv "$OLD_STYLE_FILE" "$NEW_STYLE_FILE"
+            find . -type f -exec sed -i "s/$OLD_STYLE_FILE/$NEW_STYLE_FILE/g" {} +
+          fi
+
+      - name: Commit and push changes
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)